tests/integration/utils/kube_client.go

```diff
@@ -0,0 +1,137 @@
+package utils
```

Comment:
nit: utils packages don't really describe what's in the package. This is just personal preference so feel free to ignore this but imo it's better to have more packages that describe what's in them than less packages that are more generic. Could maybe be a `utils/kiali` package and a `utils/kube` package? Or `utils/kube/kiali`?

Reply:
Created packages.